modules/command-engine/core/src/test/scala/surge/core/TestBoundedContext.scala

```diff
@@ -32,11 +33,11 @@ object TestBoundedContext {
   case class DoNothing(aggregateId: String) extends BaseTestCommand
   case class CreateNoOpEvent(aggregateId: String) extends BaseTestCommand
 
-  case class FailCommandProcessing(failProcessingId: String, withError: Throwable) extends BaseTestCommand {
```

Comment:
had to change these classes a bit .... akka (or something else ...) was attempting to serialize these Throwables and failing so I'm sending the errorMsg as a string and throwing the exception on the other side